docs/highlights/2020-10-28-aws-s3-and-firehose-sources.md

```diff
@@ -0,0 +1,34 @@
+---
+last_modified_on: "2020-10-27"
+$schema: ".schema.json"
+title: "New `aws_s3` and `aws_kinesis_firehose` sources"
+description: "Export observability data out of AWS with ease."
+author_github: "https://github.com/binarylogic"
+pr_numbers: [4101]
+release: "0.11.0"
+hide_on_release_notes: false
+tags: ["type: announcement", "domain: sources", "platform: aws"]
+---
+
+Getting observability data out of AWS can sometimes feel like you're in a
+[Rube Goldberg comic][rube_goldberg], stitching together three or more tools
```

Comment:
This feels like a good time to use "umpteen" 😄

Reply:
Excellent choice :)